backend/repopulate_db.py

```diff
@@ -3,20 +3,16 @@
 # production.
```

Comment:
This file is getting quite big. I think we should make a `backend/data folder` and store .json files for each course and then repopulate_db.py should read and process all files in the data `folder`.

Reply:
Resolved; we agreed to leave this for now as it is getting replaced in phase 2, and having some data in the json being modified in this file would make things confusing